src/server/config/keys.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  google: {
+    clientID:
+      // eslint-disable-next-line max-len
+      '294540986299-on4oith7fml2vhsumdesfr9npjr8r2um.apps.googleusercontent.com',
+    clientSecret: 'Iasf-PGRAwt6MhDZpuzI2Ems',
```

Comment:
There are secure keys. We don't need to push it into public repository. Pls create dummy keys insteal and share those values using private conference.